core/src/com/unciv/models/ruleset/unique/Countables.kt

```diff
@@ -36,6 +36,13 @@ object Countables {
         if (countable.equalsPlaceholderText("Remaining [] Civilizations"))
             return gameInfo.civilizations.filter { !it.isDefeated() }
                 .count { it.matchesFilter(placeholderParameters[0]) }
+        
+        if (countable.equalsPlaceholderText("[] Tiles")) {
+            val ownedTiles = gameInfo.tileMap.values.filter {
```

Comment:
For performance, we want to 
- iterate on civ cities
- sum the tiles for each city that fit the filter
- sum the total counts

So something like civInfo.cities.sumOf { it.tiles.count { it.matchesFilter(placeholderParameters[0]) } }